contracts/ExitHandler.sol

```diff
@@ -181,6 +181,64 @@ contract ExitHandler is DepositHandler {
     exitDuration = _exitDuration;
   }
 
+  function startLimboExit(
+    bytes memory limboTxData, bytes32 signature, uint8 _outputIndex,
```

Comment:
@nanspro i think you have been looking at the wrong specification. there should not be any signatures for more viable plasma. can you make sure that you tried to implement this: https://ethresear.ch/t/more-viable-plasma/2160